packages/block-editor/src/index.js

```diff
@@ -13,6 +13,7 @@ export {
 	getColorClassesAndStyles as __experimentalGetColorClassesAndStyles,
 	useColorProps as __experimentalUseColorProps,
 	useCustomSides as __experimentalUseCustomSides,
+	getSpacingClassesAndStyles as __experimentalGetSpacingClassesAndStyles,
```

Comment:
We also have to import this for native in https://github.com/WordPress/gutenberg/blob/e66e6d98d70b5a339f4548e67872122024e761e4/packages/block-editor/src/hooks/index.native.js

`export { getSpacingClassesAndStyles } from './use-spacing-props';`